.github/workflows/docker_build_trt.yml

```diff
@@ -0,0 +1,65 @@
+name: Build & Publish Docker Image
+# https://www.docker.com/blog/first-docker-github-action-is-here
+# https://github.com/docker/build-push-action
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+  release:
+    types: [published, edited]
+
+jobs:
+  build-runtime:
+    runs-on: ubuntu-20.04
+    strategy:
+      fail-fast: false
+      matrix:
+        runtime_type: [onnxruntime-tensorrt]
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Get release version
+        id: get_version
+        run: |
+          VER=${GITHUB_REF##*/}
+          echo "RELEASE_VERSION=$(${{ github.event_name == 'release' }} && echo "${VER//v}" || echo "$VER")" >> $GITHUB_ENV
+
+      - uses: docker/setup-buildx-action@v1
+      - name: Build and Publish Docker Image
+        uses: docker/build-push-action@v1.1.0
+        with:
+          repository: nodefluxio/vortex
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+          tags: "runtime-${{ matrix.runtime_type }}-${{ env.RELEASE_VERSION }}"
+          target: runtime
+          dockerfile: dockerfiles/onnxruntime-tensorrt.dockerfile
+          push: ${{ github.event_name == 'release' }}
+        timeout-minutes: 120
+  
+  build-development:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Get release version
+        id: get_version
+        run: |
+          VER=${GITHUB_REF##*/}
+          echo "RELEASE_VERSION=$(${{ github.event_name == 'release' }} && echo "${VER//v}" || echo "$VER")" >> $GITHUB_ENV
+
+      - uses: docker/setup-buildx-action@v1
+      - name: Build and Publish Docker Image
+        uses: docker/build-push-action@v1.1.0
+        with:
+          repository: nodefluxio/vortex
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+          tags: "${{ env.RELEASE_VERSION }},trt-latest"
```

Comment:
@triwahyuu continuing from comment, for tagging prefer to make it separate from main, so I think this liine should be
```
"development-${{ matrix.runtime_type }}-${{ env.RELEASE_VERSION }},latest"
```
or smth like that ?

Reply:
maybe this one doesn't need `latest` tag, as it would be for the main vortex development image. 
my reasoning to not include `development` in the tag is that because the image actually contains both development and runtime so I think it's the same anyway.

maybe for the vortex development with tensorrt the tag is just `${{ env.RELEASE_VERSION }}-tensorrt`, what do you think?

Reply:
ok that seems good brb change it